src/features/authService/auth.service.js

```diff
@@ -49,4 +49,39 @@ export default function ($rootScope, $http, $location, $sessionStorage, $base64,
             str.push(encodeURIComponent(p) + "=" + encodeURIComponent(obj[p]));
         return str.join("&");
     }
+
+
+    this.loginFb = function () {
+        $facebook.init();
+        $facebook.login().then(function() {
+            getFacebookUser();
+        });
+
+    }
+    function getFacebookUser() {
+        $facebook.api("/me",{fields: 'id,name,email,last_name,first_name,picture' }).then(
+            function(response) {
+                console.log(response);
+                var fbUser = {
+                    "avatarFileId": "string",
+                    "email": response.email,
+                    "firstName": response.first_name,
+                    "id": response.id,
+                    "lastName": response.last_name,
+                    "password": "string"
+                };
+                $http.post('/api/users', fbUser)
```

Comment:
Poniżej wywołujesz `login(fbUser.email,fbUser.password);` dla sukcesu i dla błędu jest to samo, czyli możesz pominąć tego posta.